docs/getting-started.md

```diff
@@ -14,11 +14,11 @@ For information around how to set up React Native, see the [React Native Getting
 Remember to call `react-native init` from the place you want your project directory to live.
 
 ```
-npx react-native init <projectName> --version ^0.62.2
+npx react-native init <projectName> --version 0.62.2

```

Comment:
This should be `npx react-native init <projectName> --template react-native@^0.62.2`

Reply:
fixed!